src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -641,7 +630,13 @@ void DistributedSearchTreeImpl<DeviceType>::communicateResultsBack(
   // We are assuming here that if the same rank is related to multiple batches
   // these batches appear consecutively. Hence, no reordering is necessary.
   Distributor<DeviceType> distributor(comm);
-  int const n_imports = distributor.createFromSends(space, ranks, offset);
+  // FIXME Distributor::createFromSends takes two views of the same type by
+  // const reference.  They were two easy way out, either take the views by
```

Comment:
English?

Reply:
This is a cheap solution that gets things to compile.

Reply:
```suggestion
  // a const reference.  There were two easy ways out, either take the views by
```